src/mono/mono.proj

```diff
@@ -321,17 +325,25 @@
     <PropertyGroup Condition="'$(TargetsBrowser)' == 'true'">
       <_MonoMinimal Condition="'$(Configuration)' == 'Release'">,debugger_agent,log_dest</_MonoMinimal>
       <_MonoMinimal Condition="'$(Configuration)' == 'Release' and '$(MonoEnableAssertMessages)' != 'true'">$(_MonoMinimal),assert_messages</_MonoMinimal>
+      <_MonoMinimal Condition="'$(MonoWasmThreads)' != 'true'">$(_MonoMinimal),threads</_MonoMinimal>
+      <_MonoMinimal Condition="'$(MonoWasmThreadsNoUser)' == 'true'">$(_MonoMinimal),wasm_user_threads</_MonoMinimal>
```

Comment:
this looks backwards, i assume ENABLE_MINIMAL=x is a list of features we're *removing*?

Reply:
> a list of features we're removing

Correct.  It ends up defining `DISABLE_whatever` in the C code.  So in this case, we have `DISABLE_THREADS` if `MonoWasmThreads` isn't set to `true`, and `DISABLE_WASM_USER_THREADS` if `MonoWasmThreadsNoUser` _is_ set to true.

The idea being that we will do a build with `!defined (DISABLE_THREADS) && defined(DISABLE_WASM_USER_THREADS)` for the "EventPipe gets threads and nobody else does" configuration.  That way all of the normal mono threading stuff is just checking for `!defined (DISABLE_THREADS)` (there's already a ton of code under this condition), and the one or two places that are in charge of starting user threads will do an additional check for `!defined (DISABLE_WASM_USER_THREADS)`.